packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/SelectedEntriesModal/index.js

```diff
@@ -390,11 +407,21 @@ const SelectedEntriesModalContent = ({
   );
 };
 
+SelectedEntriesModalContent.defaultProps = {
+  validationErrors: null,
```

Comment:
I did it, the only part where I defined the default value as `undefined` is for the `EntryValidationText` because when we pass the value we pass it like this
```
<EntryValidationText
                  validationErrors={validationErrors[row.id]}
                  isPublished={row.publishedAt !== null}
                />
```
and could be an object or undefined, then in the EntryValidationText we can just check if validationErrors is defined or not to show the messages